.bash_aliases

```diff
@@ -56,12 +71,22 @@ mk() {
 }
 mksh() { if [ "$#" -ne 1 ]; then echo "usage: mksh file"; else mk "$1" '#!/usr/bin/env bash'; fi; }
 mkpy() { if [ "$#" -ne 1 ]; then echo "usage: mkpy file"; else mk "$1" '#!/usr/bin/env python'; fi; }
+# I like these^ :D I use vim abbreviations to do it, typeing '#!b<SPACEBAR>'
```

Comment:
I honestly don't use these enough for them to be justified. I like your strategy better :/